_config.yml

```diff
@@ -4,6 +4,7 @@ volunteers_form: "https://goo.gl/forms/gIMkkXcNpJjL98t23"
 mentors_form: "https://goo.gl/forms/PYQtbtLitfD1gaYi2"
 hackers_form: https://localhackday.mlh.io/lhd-2018/events/307-university-of-colorado
```

Comment:
Hacker form should be the registration link I think
Or just delete this if we don't need it